feat(projectgridcard): create project grid card

lialila opened the pull request:
Preview: https://storybook-5uhhevzia-treely.vercel.app/?path=/story/slices-projectsgrid--minimal

Story: https://trello.com/c/6EkhLqjX/522-project-status-on-portfolio-pages-is-fetched-from-fpm-5-13

lukasbals commented:
Two things that come to my mind:

* There is no action triggered in Storybook when clicking on a card in the `ProjectsGrid` component. 
* What about renaming the `FpmProjectCard` component to `ProjectCard`?

lukasbals commented on src/components/CreditsAvailableBadge/CreditsAvailableBadge.tsx:
```diff
@@ -16,40 +17,50 @@ const CreditsAvailableBadge = ({
-    <Badge
+    <Flex
+      justifyContent="flex-start"
       as={href ? NextLink : undefined}
       href={href}
-      colorScheme={variant.color}
-      width="fit-content"
     >
```
Is this `Flex` needed?

lialila replied:
yes, to make the component as short as the text, not to stretch

lukasbals commented on src/components/CreditsAvailableBadge/messages.en.ts:
```diff
@@ -1,7 +1,7 @@
 const messagesEn = {
   'components.creditsAvailableBadge.text.yes': 'Credits available',
-  'components.creditsAvailableBadge.text.some': 'Some remaining credits',
-  'components.creditsAvailableBadge.text.no': 'No remaining credits',
```
I would keep the "old" wording here since this is what marketing accepts.

lukasbals commented on src/rootMessages.de.ts:
```diff
@@ -34,6 +33,7 @@ const rootMessagesDe = {
   ...customerCardMessagesDe,
   ...customerQuoteCardMessagesDe,
   ...glossaryMessagesDe,
+  ...projectGridCardMessagesDe,
   ...projectFactsMessagesDe,
```
```suggestion
  ...projectFactsMessagesDe,
  ...projectGridCardMessagesDe,
```

lukasbals commented on src/slices/ProjectGridCard/messages.de.ts:
Please remove if not needed

lukasbals commented on src/slices/ProjectGridCard/messages.en.ts:
Please remove if not needed

lukasbals reviewed (commented):
Good work so far! Big clean up.

lialila commented:
> Two things that come to my mind:
> 
> * There is no action triggered in Storybook when clicking on a card in the `ProjectsGrid` component.
> * Please update the commit message to reflect what you actually did

- Added `slug` to `portfolioProjectMock`, this resolved the issue. In `app` the link to project page also works.
- Updated the commit

lukasbals commented on README.md:
````diff
@@ -37,11 +37,17 @@ For local testing, you can link the package to your project:
 npm install ../path/to/strapi-slices-package.tgz
 ```
 
+Note: The Strapi CMS uses the data from the fpm-api for staging.
````
```suggestion
Note: The Strapi CMS uses the data from the `fpm-api` from staging.
```
